extensions/wikia/Forum/ForumController.class.php

```diff
@@ -313,16 +354,16 @@ public function forumActivityModule() {
 	}
 
 	public function forumRelatedThreads() {
-		$title = Title::newFromId( $this->app->wg->Title->getText() );
+		$title = Title::newFromID( $this->app->wg->Title->getText() );
 		$this->response->setVal( 'showModule', false );
 		if ( !empty( $title ) && $title->getNamespace() == NS_WIKIA_FORUM_BOARD_THREAD ) {
 
 			$rp = new WallRelatedPages();
-			$out = $rp->getMessageRelatetMessageIds( [ $title->getArticleId() ] );
-			$messages = [ ];
+			$out = $rp->getMessageRelatetMessageIds( [ $title->getArticleID() ] );
+			$messages = [];
 			$count = 0;
 			foreach ( $out as $key => $val ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$key" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)